.circleci/config.yml

```diff
@@ -0,0 +1,111 @@
+    description: "Install system dependencies"
+    steps:
+      - run:
+          name: install system dependencies
+          command: |
+            # The CircleCI environment is based on Debian buster, which used to be called stable Debian, but is now called oldstable.
+            wget -q -O - https://dl.google.com/linux/linux_signing_key.pub | sudo apt-key add -
+            sudo apt-get update --allow-releaseinfo-change && \
+            sudo apt-get update && \
+            sudo apt-get install -y pdftk ghostscript
+  install_js_dependencies:
+    description: "Install JS dependencies"
+    steps:
+      - restore_cache:
+          key: yarn-{{ checksum "yarn.lock" }}
+      - run:
+          name: yarn install --frozen-lockfile
+          command: yarn install --frozen-lockfile
+      - save_cache:
+          key: yarn-{{ checksum "yarn.lock" }}
+          paths:
+            - /mnt/ramdisk/node_modules
+            - ~/.npm
+  install_ruby_dependencies:
+    description: "Install Ruby dependencies"
+    steps:
+      - restore_cache:
+          key: bundle-{{ checksum "Gemfile.lock" }}
+      - run:
+          name: install bundler & ruby dependencies
+          command: |
+            gem install bundler:2.3.5 --no-document && \
+            bundle install --jobs=4 --retry=3 --path vendor/bundle
+      - save_cache:
+          key: bundle-{{ checksum "Gemfile.lock" }}
+          paths: [/mnt/ramdisk/vendor/bundle]
+  setup_test_db_for_parallel:
+    description: "Setup test database"
+    steps:
+      - run: dockerize -wait tcp://localhost:5432 -timeout 1m
+      - run: bundle exec rake db:create db:schema:load
+  setup_test_db:
+    description: "Setup test database"
+    steps:
+      - run: dockerize -wait tcp://localhost:5432 -timeout 1m
+      - run: bundle exec rake db:create db:schema:load
+  setup_webdriver:
+    description: Install Ruby webdriver
+    steps:
+      - run: date -I > /tmp/today && cat /tmp/today
+      - restore_cache:
+          key: webdrivers-v2-{{ checksum "/tmp/today" }}
+      - run: if [ ! -f ~/.webdrivers/chromedriver ] ; then bundle exec rake webdrivers:chromedriver:update ; fi
+      - save_cache:
+          key: webdrivers-v2-{{ checksum "/tmp/today" }}
+          paths: [~/.webdrivers]
+jobs:
+  i18n_normalize:
+    executor: ruby_executor
+    steps:
+      - checkout
+      - install_ruby_dependencies
+      - run: bundle exec i18n-tasks normalize
+      - run: git config --global user.name CircleCI && git config --global user.email circleci@example.com
+      - add_ssh_keys:
+          fingerprints:
+            - "0a:b8:14:ae:05:dd:17:9c:f4:6e:3c:45:81:8f:06:8d"
+      - run: git diff --quiet || (git commit -am "Run i18n-tasks normalize" && git push origin translations_config-locales-en-yml--main_es)
+  run_ruby_tests:
+    executor: rails_executor
+    steps:
+      - checkout
+      - install_system_dependencies
+      - install_ruby_dependencies
+      - setup_test_db_for_parallel
```

Comment:
Since you're doing `bundle exec rspec` rather than `bundle exec parallel_rspec`, I think you should rename `setup_test_db_for_parallel` into `setup_test_db`.

Reply:
Resolved in 40538b5